app/serializers/cohort_csverizer.rb

```diff
@@ -0,0 +1,11 @@
+class CohortCsverizer < ActiveModel::Csverizer
+  attributes :description, :twitter_ids, :latest_data
+
+  def twitter_ids
+    object.twitter_ids.join(',')
+  end
+
+  def latest_data
+    DataSetSerializer.new(object.latest_data_set).to_json
+  end
```

Comment:
Is this intended to be the DataSetSerializer and not the DataSetCsverizer?

Reply:
`DataSetSerializer` is fine here, we're going to return a json data object as a csv column.